brails/workflow/Images.py

```diff
@@ -14,7 +14,22 @@
 from multiprocessing.dummy import Pool as ThreadPool
 import requests 
 from pathlib import Path
+from functools import lru_cache
 
+@lru_cache(maxsize=None)
```

Comment:
The `@lru_cache` decorator automatically creates a cache for API keys so that the validation process will only be run the first time the function is called. This will be valuable when we later add more extensive checks which may require running requests, etc.